### DISCUSSION TITLE
How do I start calendar?

### DISCUSSION THREAD

**Discussion by hardfau1t** [nvim-neorg/neorg#877] [Q&A] [answered]
I checked the documentation of calendar module and added `["core.ui.calendar"] = {}` to neorg config but I don't see anyway to open the calendar, Am I missing something?

**Comment by ldmax** [comment]
Same here. I'm really excited to see new releases with information about calendar but don't have a clue about how to start calendar.
Please can someone help?

**Comment by inhake2** [comment, marked as answer]
You should be able to see and use the (very nice !) calendar ui with the `<localleader>id` key.
If you use the journal module, you can use it with `:Neorg journal custom` to open/create an entry.

It seems that there is no other functionality now but plenty of possibilites for the future.
I’d like to know how to use the calendar API to play with it. Should dig into neorg code, maybe !
